modules/kueezRtbBidAdapter.js

```diff
@@ -176,6 +177,12 @@ function buildRequests(validBidRequests, bidderRequest) {
   const bidderTimeout = bidderRequest.timeout ?? config.getConfig('bidderTimeout');
   const requests = [];
   validBidRequests.forEach(validBidRequest => {
+    try {
+      if (validBidRequest.params && (validBidRequest.params.mode === 'LATEST' || validBidRequest.params.mode !== 'BASIC')) {
+        loadExternalScript('https://static.kueezrtb.com/latest.js', BIDDER_CODE, () => {});
```

Comment:
sorry, this is not allowed

Reply:
@patmmccann what is the issue?
According to the guidelines, if there is a disclosure and it can be opted-out, it should be ok.